migrations/00000-createTableGuests.ts

```diff
@@ -0,0 +1,29 @@
+import { Sql } from 'postgres';
+import { z } from 'zod';
+
+export type Guest = {
+  id: number;
+  firstName: string;
+  lastName: string;
+  attending: boolean;
+};
+
+export const guestsSchema = z.object({
+  firstName: z.string().min(1).max(30),
+  lastName: z.string().min(1).max(30),
+});
```

Comment:
is this missing "attending"?

Reply:
looks like this is done in a different PR:

- https://github.com/upleveled/expo-example-spring-2024-atvie/pull/7